games-emulation/dolphin/dolphin-5.0_p20220520.ebuild

```diff
@@ -35,12 +35,14 @@ IUSE="
 	profile pulseaudio systemd upnp vulkan
 "
 
+PATCHES=("${FILESDIR}/${PN}-libfmt-9.0.0.patch")
+
 RDEPEND="
 	app-arch/bzip2:=
 	app-arch/xz-utils:=
 	app-arch/zstd:=
 	dev-libs/hidapi:=
-	=dev-libs/libfmt-8*:=
+	>=dev-libs/libfmt-8:=
```

Comment:
needs a revbump to unpin

Reply:
Sorry, noticed this too late, it doesn't when there's `:=`, it's technically /always/ pinned and the rebuild attempt forces to check the ebuild. i.e. let you upgrade libfmt again and cause a rebuild.

To be sure, gave it a quick try while also ensuring dynamic-deps is off

While still pinned, as expected:
```
# emerge --dynamic-deps=n -uUDav @world
<snip>
WARNING: One or more updates/rebuilds have been skipped due to a dependency conflict:
dev-libs/libfmt:0
```

Reverted to `>=` but without a revbump:
```
# emerge --dynamic-deps=n -uUDav @world
<snip>
The following packages are causing rebuilds:

  (dev-libs/libfmt-9.0.0:0/9.0.0::gentoo, ebuild scheduled for merge) causes rebuilds for:
    (games-emulation/dolphin-5.0_p20220520:0/0::gentoo, ebuild scheduled for merge)
```

Reply:
Albeit people will rebuild anyway for the libfmt upgrade, so I it's not a big deal in ~arch, but wanted to mention for future reference.

Reply:
I don't think there's an obligation for Portage to act that way though. It could have just as easily checked the VDB, seen its recorded dep had =dev-libs/libfmt-8*, and because it doesn't match libfmt, skipped the upgrade. 

The "will Portage check the VDB or check the ebuild" issue doesn't really have a solid answer. Ofc, to actually do the rebuild, it'd have to re-read the ebuild, but I don't think there's anything preventing it from not considering it in the first place because it has a dep which precludes it?

In any case, even if Portage does act that way, it doesn't seem guaranteed and is nth order reasoning about the VDB which isn't a good idea.

Reply:
Then again, I guess it would matter if the pinning had been more specific, e.g. =libfmt-8.1.1 given this affects multiple of the same subslot. So maybe revbump isn't so bad to be safe rather than consider individual situations.

Reply:
Yeah, giving it more thought and looking at VDB, you're right. Thanks.